### PR TITLE
Week2 homework files for JS BitCamp

### PR DESCRIPTION
**Pull request by danzelo1**

### CODE REVIEW

File: Javascript/homework/responses/2.1-css-linking.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+*In this week's assignment, students will build off of last weeks work and style their existing webpage to produce a more aesthetically pleasing digital clock.*
+
+#### Task 1: Linking your CSS file
+
+This week, we're going to take the HTML and JavaScript code we wrote last week and style our page using CSS to make it more aesthetically pleasing.  First, copy your index.html file into a new html file in your week 2 homework folder.
+
+To style our webpage, we must link a css file to our HTML file.  To do this, add a `link` tag inside the head of your html file. This tag should have the `rel` attribute set to `"stylesheet"` and the `href` attribute set to `"style.css"`.
+
+Next, open a new file in your week 2 folder and save it as `style.css`. It's important that your file name matches the one you typed in your HTML document in its link tag's `href` attribute, as this will be what links your two files.
+
```
Comment (MinyMeep1) [inline Javascript/homework/responses/2.1-css-linking.md]: I personally see no issue. And if someone doesn't know how to write the code properly maybe we can include it. But it's simply googling how to href a file.

File: Javascript/homework/responses/2.2-body.md [hunk 2]
```diff
@@ -4,4 +4,9 @@
 
 #### Task 2: Styling the body
 
-In the body class, center all items, select a font and size for your text, and change the background color of the page.
\ No newline at end of file
+In your CSS file, define a body class and set the display to flex.  This will make sure all elements fit the screen appropriately. Next, use the `justify-content`  and `align-items` attributes to center your items on the y axis and the x axis respectively. You may also want to set the `min-height` to `100vh` so that height of your content will always be relative to the size of the screen.
```
Comment (MinyMeep1) [inline Javascript/homework/responses/2.2-body.md]: How do you define a body class? Should this be googled as well to display to flex?

Comment (MinyMeep1) [inline Javascript/homework/responses/2.2-body.md]: This is all self-explanatory. Unless we need to explain what a `justify-content` is or have them google the exact sheet.

File: Javascript/homework/responses/2.3-clock-design.md [hunk 3]
```diff
@@ -4,5 +4,21 @@
 
 #### Task 3: Creating your clock
 
-Also define a clock class - this is where we will design a digital clock to print the time.
+After you're satisfied with the body of your page, define a block that styles elements in the `clock` class.
```
Comment (MinyMeep1) [inline Javascript/homework/responses/2.3-clock-design.md]: How do you "define a block that styles elements in the `clock` class." unclear.

File: Javascript/homework/responses/2.3-clock-design.md [hunk 4]
```diff
@@ -4,5 +4,21 @@
...
+
+![Screen Shot 2021-01-15 at 5.49.29 PM](/Users/danielleanzelone/Library/Application Support/typora-user-images/Screen Shot 2021-01-15 at 5.49.29 PM.png)
+
+First, set a width and a height for the box that's going to hold your time.  Add a background and a solid border using colors different from the screen's background so you can see what the box will look like.
+
+Once again, you'll want to set your `display` to `flex` and center your text using `align-items` and `justify-content` .  You need to do this again because these attributes in the body only centered the box on your page; repeating these lines in the clock section will center your text within the box.
```
Comment (MinyMeep1) [inline Javascript/homework/responses/2.3-clock-design.md]: What is the display or flex? The question stands should we have students google this or explain every detail.

File: Javascript/homework/responses/2.3-clock-design.md [hunk 5]
```diff
@@ -4,5 +4,21 @@
...
+
+You may also want to adjust the margins of your clock so that theres some space between the box and the form - I simply adjusted the left margin of my clock.
+
+You can add as many bells and whistles to this clock that you please.  I increased the font-size here and added a box-shadow to add some depth to my clock.
+
+![Screen Shot 2021-01-15 at 6.17.26 PM](/Users/danielleanzelone/Library/Application Support/typora-user-images/Screen Shot 2021-01-15 at 6.17.26 PM.png)
+
+The first number in each line represents the x-offset which is the horizontal distance from which you want the object to appear.  The second number is the y-offset - the vertical distance from which the box will appear.  The third number is the blur radius - a larger blur radius will create a bigger shadow and vice versa. 
+
+Lastly is the color.  I used the `rgba()` tool to define my color.  The first 3 numbers represent the amount of red, green, and blue in the color, respectively. The fourth number is the opacity on a scale from 0 to 1, with 0 being complete transparency and 1 being fully opaque.
```
Comment (MinyMeep1) [inline Javascript/homework/responses/2.3-clock-design.md:26]: What is the rgba tool and how do we use this?

Comment (MinyMeep1) [reply Javascript/homework/responses/2.3-clock-design.md:26]: "The first 3 numbers represent the amount of red, green, and blue in the color, respectively." This explains it but not fully or how to open it.

File: Javascript/homework/responses/2.4-form-layout.md [hunk 6]
```diff
@@ -4,4 +4,8 @@
 
 #### Task 4: Styling your buttons 
 
-Finally, define a timezone class to style the labels and buttons used for your timezone selector.
\ No newline at end of file
+Lastly, you'll define a timezone class to style the labels and buttons used for your timezone selector.  Similarly to the way you defined the clock styles using the unique id from the clock div, preceded by a '.', you'll define the radio button style class using whatever class you used to group together your timezone buttons. I grouped my radio buttons and labels in a div with the class set to `"timezone"` so my CSS file looks like this:
```
Comment (MinyMeep1) [inline Javascript/homework/responses/2.4-form-layout.md]: Pretty good with this.

**Review by emsesc — CHANGES_REQUESTED** [review]
Same issues as Week 4:
1. README.md file format was [updated](https://github.com/bitprj/BitCamp/tree/master/repo_logistics/sample-camp)
2. More bullet points than paragraph formatting would make it more readable
3. Make sure titles (h1/h4 tags) adhere to what is [here](https://github.com/bitprj/BitCamp/tree/master/repo_logistics) (Also add a description for each week's steps!)
4. Fix images by uploading them to a /images directory in /responses `/responses/images`

**Review by emsesc — APPROVED** [review]

**Comment by emsesc** [comment]
@danzelo1 Please add a italicized quick description of the course in `course-deatils.md` under the title. Refer to [this](https://github.com/emsesc/sample-learninglab/blob/main/course-details.md) for an example. Also, go ahead and resolve the conflicts. Thanks!

**Pull request closed without merging**